problems/variables.js

```diff
@@ -31,7 +44,11 @@
 //  * Convert it to fahrenheit and output "`tempInCelsius`°C is `tempInFahrenheit`°F".
 //  * Now store a fahrenheit temperature into a variable.
 //  * Convert it to celsius and output "`tempInFahrenheit`°F is `tempInCelsius`°C."
-
+// var tempInCelsius = 32;
+// var tempInFahrenheit = (32 * 9/5) + 32;
+// console.log(tempInCelsius + " is " + tempInFahrenheit);
+// var tempInCelsius = (89.6 - 32) * 5/9;
```

Comment:
You used 'tempInCelsius' twice (line 47 & 50). They should be named two different things for clarity.